### PR TITLE
Fix bad param name for schedule_screen

### PR DESCRIPTION
**Pull request by jkonecny12**
All other schedule methods have `ui` only this one has `ui_screen`.

### CODE REVIEW

File: simpleline/render/screen_scheduler.py [hunk 1]
```diff
@@ -91,17 +91,17 @@ def nothing_to_render(self):
         """
         return self._screen_stack.empty()
 
-    def schedule_screen(self, ui_screen, args=None):
+    def schedule_screen(self, ui, args=None):
```
Comment (poncovka) [inline simpleline/render/screen_scheduler.py]: I think that `ui_screen` is more readable.

Comment (jkonecny12) [inline simpleline/render/screen_scheduler.py]: Fine, why not. I'm going to do opposite change in this PR :D

Comment (jkonecny12) [inline simpleline/render/screen_scheduler.py]: Fixed

**Comment by jkonecny12** [comment]
This commit was added to PR https://github.com/rhinstaller/python-simpleline/pull/11 because that PR depends on changes which are here.
Closing this one.

**Pull request closed without merging**